mobile/src/components/Menu.js

```diff
@@ -33,6 +35,10 @@ const styles = {
     color: '#337ab7',
     paddingLeft: 5,
   },
+  lastOnlineText: {
```

Comment:
Can you move all this component file styling to separate styling file?

Reply:
We can but I think it would require refactor all other components. Perhaps there is a ticker for it already.

Reply:
ok